src/main/resources/META-INF/plugin.xml

```diff
@@ -50,6 +50,7 @@
 
     <change-notes><![CDATA[
       <ul>
+        <li>4.8 - Support of rules parameters configuration for IntelliJ. Fewer false positives and performance improvement for PHP. 4 new vulnerability detection rules for Python.</li>
```

Comment:
```suggestion
        <li>4.8 - Support offline configuration of rule parameters. Performance improvement and fewer false positives for PHP. 4 new vulnerability detection rules for Python.</li>
```

Reply:
@christophelevis could you please have a look at the suggested release notes for SLI 4.8?